.github/workflows/gateway-conformance.yml

```
@@ -0,0 +1,146 @@
+          echo "slug=$slug" | tee -a $GITHUB_OUTPUT
+
+      # 2. Run the gateway-conformance tests
+      - name: Run gateway-conformance tests
+        uses: singulargarden/gateway-conformance/.github/actions/test@main
+        with:
+          gateway-url: ${{ matrix.gateway_url }}
+          json: output.json
+          xml: output.xml
+          html: output.html
+          markdown: output.md
+          report: report.json
+          accept-test-failure: true
+
+      # 3. Upload the results
+      - name: Upload MD summary
+        # TODO: generate a minimal output.md in the action
+        if: failure() || success()
+        run: cat output.md | sed '/Failures\/Errors/,$d' >> $GITHUB_STEP_SUMMARY
+      - name: Upload JSON output
+        if: failure() || success()
+        uses: actions/upload-artifact@v3
+        with:
+          name: conformance-${{ steps.slug.outputs.slug }}.json
+          path: |
+            ./output.json
+            ./output.html
+            ./report.json
+      - name: Upload HTML report
+        if: failure() || success()
+        uses: actions/upload-artifact@v3
+        with:
+          name: conformance-${{ steps.slug.outputs.slug }}.html
+          path: output.html
+
+  aggregate:
+    permissions:
+      contents: write
+    runs-on: "ubuntu-latest"
+    needs: [conformance]
+    # the tests might have failed
+    if: always()
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v3
+      - name: Download Artifacts
+        uses: actions/download-artifact@v3
+        with:
+          path: artifacts
+      - name: Aggregate reports
+        run: |
+          mkdir ./reports
+
+          # download-artifact downloads artifacts in a directory named after the artifact
+          # details: https://github.com/actions/download-artifact#download-all-artifacts
+          for folder in ./artifacts/conformance-*.json; do
+            file="${folder}/report.json"
+            new_name="${folder#.\/artifacts\/conformance-}" # drop path prefix "./artifacts/conformance-"
+            new_file="./reports/${new_name}"
+            cp "${file}" "${new_file}"
+          done
+      - name: Upload Data Aggregates
+        # This will be useful for local debugging
+        if: (failure() || success())
+        uses: actions/upload-artifact@v3
+        with:
+          name: dashboard-reports
+          path: ./reports
+      - name: Generate final report
+        run: |
+          node ./.github/aggregate.js ./reports/*.json > ./src/report.json
+      - name: Upload Report
+        # This will be useful for local debugging
+        if: (failure() || success())
+        uses: actions/upload-artifact@v3
+        with:
+          name: dashboard-report
+          path: ./src/report.json
+      - name: Configure git
+        run: |
+          git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com>"
+          git config --global user.name "${GITHUB_ACTOR}"
```

I think we might want to use the bot user creds here after all. From experience with GitHub Management, it's a bit confusing when CI makes a push and it's associated with a real user. I updated the config setting in GitHub Management to what I'm proposing here too. The values I propose will associate the commits with GitHub Actions user.

```suggestion
          git config user.name "github-actions[bot]"
git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

---

That's really cool, thanks for proposing the change and sharing a ready-to-commit suggestion.